### PR TITLE
#0 - update deps

### PR DESCRIPTION
**Pull request by olenagerasimova**
Update asto, http, http-client and ppom.

### CODE REVIEW

File: src/test/java/com/artipie/pypi/http/ProxySliceTest.java [hunk 1]
```diff
@@ -108,7 +120,10 @@ void getsFromCacheOnError(final String data, final String header, final String k
             new SliceHasResponse(
                 Matchers.allOf(
                     new RsHasStatus(RsStatus.OK), new RsHasBody(body),
-                    new RsHasHeaders(new Header("content-type", header))
+                    new RsHasHeaders(
+                        new MapEntry<>("content-type", header),
+                        new MapEntry<>("Content-Length", size)
+                    )
```
Comment (genryxy) [inline src/test/java/com/artipie/pypi/http/ProxySliceTest.java:126]: @olenagerasimova probably do not specify size in csv source and get size from data

File: src/test/java/com/artipie/pypi/http/ProxySliceTest.java [hunk 2]
```diff
@@ -149,15 +164,25 @@ void returnsNotFoundWhenRemoteReturnedBadRequest() {
...
                 new FromRemoteCache(this.storage)
             ),
             new SliceHasResponse(
-                Matchers.allOf(new RsHasBody(body), new RsHasHeaders(header)),
+                Matchers.allOf(
+                    new RsHasBody(body),
+                    new RsHasHeaders(
+                        new MapEntry<>("content-type", "smth"),
+                        new MapEntry<>("Content-Length", "15")
+                    )
```
Comment (genryxy) [inline src/test/java/com/artipie/pypi/http/ProxySliceTest.java:184]: @olenagerasimova and here get size from body

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova please see two minor comments

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good to me

**Comment by 0crat** [comment]
Job `gh:artipie/pypi-adapter#151` is not assigned, can't get performer

**Comment by olenagerasimova** [comment]
@rultor release, tag=`0.7.4`

**Pull request merged**